netexec.cheat

```diff
@@ -18,7 +18,7 @@ nxc smb <ip> -u 'anonymous' -p '' --local-auth
 # SMB Share Guest session
 nxc smb <ip> -u 'guest' -p ''
 # SMB Share Authenticated session
-nxc smb <ip> -u '<username>' -p '<password>'
+nxc smb <ip> -u '<username>' -p '<password>'  # or -H '<hash>'
```

Comment:
Not sure if the comment at the end of a line like this works... I'd have to test this locally w/ Navi. If it doesn't the suggested change is:

```suggestion
# SMB Shared Authenticated session (user/pass)
nxc smb <ip> -u '<username>' -p '<password>'

# SMB Shared Authenticated session (Pass the Hash)
nxc smb <ip> -H '<hash>'
```